src/components/CheatSheets/CheatSheet.jsx

```diff
@@ -51,48 +73,62 @@ const CheatSheet = ({ data, heading }) => {
-                                            <RoadmapDetails key={id}>
-                                                <HackBookHeading>
-                                                    {title.replace(/[^a-zA-Z0-9]+/g, " ")}
-                                                </HackBookHeading>
-                                                {hackBookData[title].map(
-                                                    (file, fileId) =>
-                                                        Object.keys(file)[0].endsWith(".md") && (
-                                                            <RoadmapDetailsCard
-                                                                key={fileId}
-                                                                isSelected={
-                                                                    selectedFile &&
-                                                                    selectedFile.name === Object.keys(file)[0]
-                                                                }
-                                                                onClick={() =>
-                                                                    handleFileClick(title, Object.keys(file)[0])
-                                                                }
-                                                            >
-                                                                {
-                                                                    Object.keys(file)[0]
-                                                                        .replace(/.md/g, "")
-                                                                        .replace(/[^a-zA-Z0-9]+/g, " ")
-                                                                    // .toLowerCase()
-                                                                }
-                                                            </RoadmapDetailsCard>
-                                                        ),
-                                                )}
-                                            </RoadmapDetails>
-                                        ),
-                                )}
-                            </ContentNavData>
+                            <HackBookTitleContainer>
+                                <HackBookTitle> {heading} </HackBookTitle>
+                                <ResponsiveToggleButton onClick={() => setIsContentNavVisible(!isContentNavVisible)}>
+                                    {isContentNavVisible ? (
+                                        <LuPanelTopClose fontSize={"24px"} />
+                                    ) : (
+                                        <LuPanelTopOpen fontSize={"24px"} />
+                                    )}
+                                </ResponsiveToggleButton>
+                            </HackBookTitleContainer>
+                            {isContentNavVisible && (
```

Comment:
Could you find the reason that on default this section is closing right away without clicking on the button?

https://github.com/thecyberworld/TheCyberHUB/assets/76536506/eab81d36-90a2-43bd-a080-d62bd6b8b3c1

Reply:
The reason for this, was that initially the code was `setIsContentNavVisible(window.innerWidth >= 800);` so whenever the screen becomes small it defaults to **false**, so it can be changed to 
`if (window.innerWidth >= 800) {
                setIsContentNavVisible(true);
                setToggle(true);}` , so the first thing we see on mobile screens is the index.